crd-generator/collector/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>crd-generator-parent</artifactId>
+    <groupId>io.fabric8</groupId>
+    <version>6.13-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>crd-generator-collector</artifactId>
+  <name>Fabric8 :: CRD generator :: Collector</name>
+
+  <packaging>jar</packaging>
+
+  <properties>
+    <jandex.version>3.1.7</jandex.version>
```

Comment:
I see jandex property defined in parent pom. 
https://github.com/fabric8io/kubernetes-client/blob/e3fa9c82a2c5a5ebdbca41334a2181a5b4df1253/pom.xml#L168
Can we reuse this property here ?

Reply:
Seems possible. The jandex maven plugin and core module are released together. 

Should I change the property name to `jandex.version` and move it to "Core versions" section?

Reply:
Sounds good to me